CMakeLists.txt

```diff
@@ -131,4 +132,8 @@ configure_lit_site_cfg(
   ${CMAKE_CURRENT_SOURCE_DIR}/lit.cfg.py
 )
 
-ie_cpack(BENCHMARK_APP)
+if(BUILD_PLAIDML_DEVKIT)
+  include(pml_packaging)
+else()
+  ie_cpack(BENCHMARK_APP)
+endif()
```

Comment:
This is a clever workaround, I like it. The downside though is that we might need to have multiple builds in CI. I can take a stab at some point to integrate to OpenVINO differently, which might remove the need for this.